docs/spec/v1/kustomizations.md

```diff
@@ -169,6 +169,46 @@ kustomize.toolkit.fluxcd.io/prune: disabled
 For details on how the controller tracks Kubernetes objects and determines what
 to garbage collect, see [`.status.inventory`](#inventory).
 
+### Deletion policy
+
+`.spec.deletionPolicy` is an optional field that is allowing control over
+garbage collection when a Kustomization object is deleted. The default behavior
+is to mirror the value of [`.spec.prune`](#prune).
+
+Valid values include:
+
+- `Delete` - Ensure the managed resources are deleted before the Kustomization
+   is finalized.
+- `Orphan` - Leave the managed resources when the Kustomization is deleted.
```

Comment:
We need to add MirrorPrune first and explain what it does.

Reply:
Made an attempt. Please take a look.